index.html

```diff
@@ -21,42 +23,44 @@
 
 
 					<a href="tel:(12345)67890" class="connection">
-						<img src="./images/icons/phone.svg" alt="(12345)67890">
+						<img src="./images/icons/phone.svg" alt="tel:(12345)67890">
 						(12345)67890
 					</a>
 				</div>
 				<div class="header-right">
 
-					<a href="#">English<img src="./images/icons/akar-icons_chevron-down.svg"
-							alt="akar-icons_chevron-down"></a>
+					<a href="#">English<img src="./images/icons/arrow.svg"
+							alt="arrow"></a>
 
-					<a href="#">USD<img src="./images/icons/akar-icons_chevron-down.svg"
-							alt="akar-icons_chevron-down">
+					<a href="#">USD<img src="./images/icons/arrow.svg"
+							alt="arrow">
 					</a>
-					<a href="#">Login<img src="./images/icons/carbon_user.svg" alt="carbon_user"></a>
-					<a href="#">Wishlist<img src="./images/icons/uil_heart-alt.svg" alt="uil_heart-alt"></a>
+					<a href="#">Login<img src="./images/icons/login.svg" alt="login"></a>
+					<a href="#">Wishlist<img src="./images/icons/heart.svg" alt="heart"></a>
 
 
 
 					
 				<button>
-						<img src="./images/icons/Group.svg" alt="Group">
+						<img src="./images/icons/cart.svg" alt="cart">
 				</button>
 				</div>
 			</div>
 		</div>
 		<div class="header-bottom">
-			<div class="container">
```

Comment:
Why did you do it? We have a class container in our CSS, which we're using across all sections. And here you just change name to the nav-bar and declare new styles for it at CSS, that equal to our container class. Why?!

Bring back container class